lib/NUPACK/emscripten/FullEnsemble.cpp

```diff
@@ -0,0 +1,71 @@
+#include "FullEnsemble.h"
+#include "EmscriptenUtils.h"
+
+#include "src/thermo/utils/pfuncUtilsConstants.h"
+#include "src/thermo/utils/pfuncUtilsHeader.h"
+#include "src/shared/utilsHeader.h"
+#include "src/thermo/utils/DNAExternals.h"
+#include <vector>
+#include <utility>
+#include <string> 
+
+
+FullAdvancedResult* FullEnsembleNoBindingSite(const std::string& seqString, int temperature, float kcal_delta_range_mfe_subopt, bool const pseudoknotted = false)
+{
+
+    auto autoSeqString = MakeCString(seqString);
+    char* string = autoSeqString.get();
+
+    int seqNum[MAXSEQLENGTH+1];
+    int tmpLength = strlen(string);
+    dnaStructures suboptStructs = {NULL, 0, 0, 0, 0};
+
+    convertSeq(string, seqNum, tmpLength);
```

Comment:
Maybe add a comment to clarify what this function does? Took me a minute to figure it out